local/artena/lang/en/local_artena.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
This language file ensures that a correct menu entry is created for the local plugin. Doesn't break plugin without it, just makes it a bit more user friendly and tidy. All files need reversioning.